docs: add rfc process (#1)
moxystudio/rfcs-oss: README.md

# MOXY OSS RFCs

> This repository is intended to be used for reviewing submitted RFCs related to additions or changes to MOXY OSS projects and processes.

Reviewer: ```suggestion
This repository is intended to be used for reviewing submitted RFCs (request for comments) related to additions or changes to MOXY OSS projects and processes.
```
I think it is better to present the initials in the first occurrence in the text

> The MOXY OSS RFC (request for comments) process is intended to provide a consistent way for MOXY to facilitate the transparency of decisions made for features and changes in OSS related projects and processes.

Reviewer: ```suggestion
The MOXY OSS RFC process aims to provide a consistent way for MOXY to facilitate the transparency of decisions made for features and changes in OSS related projects and processes.
```

In short, an MOXY RFC is a proposal for a significant change or a new feature.

The RFC process will:

- Facilitate the transparency of decision making by MOXY to end-users, including MOXY members themselves and the community.
- Enable individual contributors to help make decisions.
- Allow domain experts to have input in decisions.
- Manage the risk of decisions made.
- Have a snapshot of context for the future.

> 

Reviewer: I think we should add two points here:

- Triage future work according to its relevance
- Enable design decisions to be taken before the implementation stage

Author: Agree with first point. However the second one is kind of already mentioned, e.g.: `Allow domain experts to have input in decisions.` and `Enable individual contributors to help make decisions.`.

- An RFC may be accepted at the close of its final comment period. The RFC's associated pull request will be merged (with squash), at which point the RFC will become part of this repository.

## Reviewing RFCs

Members of the MOXY team will attempt to review some set of open RFC pull requests on a regular basis. If a MOXY team member believes an RFC PR is ready to be accepted, they can approve the PR using GitHub's review feature to signal their approval of the RFC.

When applicable, reviewers should use GitHub's suggestions feature to make it easier to incorporate changes to the document.

## Acceptance or dismissal of RFCs

Ideally, the RFC should gather consensus and culminate in a collaborative decision. However, when a collaborative decision is not possible, people with different stakes have different decision power. As an example, authors of the project the RFC relates to have more weight than others. MOXY's CTO and Engineer Leads may also express their sentiment to help make a final decision.

## Details on accepted RFCs

> Once an RFC is accepted then authors may implement it and submit the feature as a pull request to the associated repository. 'Accepted' is not a rubber stamp, and in particular still does not mean the feature will ultimately be merged; it does mean that the core team has agreed to it in principle and are amenable to merging it.

Reviewer: ```suggestion
Once an RFC is accepted, the authors may implement it and submit the feature as a pull request to the associated repository. 'Accepted' is not a rubber stamp, and in particular still does not mean the feature will ultimately be merged; it does mean that the core team has agreed to it in principle and are amenable to merging it.
```
